frontend/device/devicelistener.lua

```diff
@@ -51,24 +49,38 @@ end
 function DeviceListener:onShowWarmth(value)
     local powerd = Device:getPowerDevice()
     if powerd.fl_warmth ~= nil then
-        UIManager:show(Notification:new{
-            text = T(_("Warmth set to %1."), powerd.fl_warmth),
-            timeout = 1.0,
-        })
+        if powerd.fl_warmth_max == nil then
```

Comment:
This codepath is never reached. All devices with powerd.fl_warmth have powerd.fl_warmth_max too (the android has it since this very PR).

Reply:
Thank you for the clarification. This check was just for safety, because I am not familiar with the Kobo-stuff at all. 
We can drop it.